chevah_build

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/env bash
+#
+# Chevah Build Script for CPython distribution.
+#
+
+PYTHON_UPSTREAM_VERSION='2.7.8'
+PYTHON_BUILD_VERSION='.c1'
+LIBFFI_UPSTREAM_VERSION='3.2.1'
+
+# Import shared code.
+. ./functions.sh
+
+# List of OS packages required for building Python.
+UBUNTU_PACKAGES="gcc make libssl-dev zlib1g-dev m4 texinfo"
+RHEL_PACKAGES="gcc make openssl-devel zlib-devel m4 texinfo"
+SLES_PACKAGES="gcc make libopenssl-devel zlib-devel m4 texinfo"
+# List of OS packages requested to be installed by this script.
+INSTALLED_PACKAGES=''
+# For the moment, we don't install anything on OS X, Solaris, AIX and
+# unsupported Linux distros. The build requires a C compiler, GNU make, m4,
+# makeinfo (from texinfo, optional) and the header files for OpenSSL and zlib.
+
+PYTHON_VERSION=$PYTHON_UPSTREAM_VERSION$PYTHON_BUILD_VERSION
```

Comment:
We have boht + and - delimiter... so for now we are stuck with this.

The - delimiter is used to signal testing version and we can skip these version when doing the mirror and also clean them after a while.... Versions with + delimiter should be kept almost forever